CONTRIBUTING.md

```diff
@@ -32,6 +32,24 @@ Pull requests are also welcome on GitHub: https://github.com/puppetlabs/bolt
 As with other open-source projects managed by Puppet, you must digitally sign the Contributor
 License Agreement before we can accept your pull request: https://cla.puppet.com
 
+**If this is your first time submitting a PR**:
+1. Fork the Bolt project (button in the top-right, next to 'star' and 'watch')
+2. Clone your fork of Bolt
+3. Add the puppetlabs repo as an upstream - `git remote add puppet git@github.com:puppetlabs/bolt`
+4. Make a new branch off of master - `git checkout -b mybranchname`
```

Comment:
`git fetch remote && git checkout remote/master && git checkout -b mybranchname`

4.5. Add commits to your branch?